### DISCUSSION TITLE
Unhandled kernel message from a widget, No such comm

### DISCUSSION THREAD

**Discussion by jshudson** [microsoft/vscode-jupyter#15076] [Questions and Answers]
I'm having an issue with running widgets using display or interact.  The widgets display when running a cell, but interacting with them gives an error message:

No such comm: d34adf99764f44519f1a8e7f42e1773e

The jupyter output is as follows:

3:09:25.208 [debug] Cell 3 executed with state Success
13:09:25.254 [debug] Widget Message: WidgetManager: Loading class LayoutModel:@jupyter-widgets/base:2.0.0
13:09:25.255 [debug] Version of IPyWidgets 8 determined after 0.001s
13:09:25.256 [debug] Widget Message: WidgetManager: Loading class SliderStyleModel:@jupyter-widgets/controls:2.0.0
13:09:25.270 [debug] Widget Message: WidgetManager: Loading class IntSliderModel:@jupyter-widgets/controls:2.0.0
13:09:25.272 [debug] Widget Message: Received display data message d34adf99764f44519f1a8e7f42e1773e
13:09:25.286 [debug] Widget Message: Waiting for model to be available before rendering it d34adf99764f44519f1a8e7f42e1773e
13:09:25.286 [debug] Widget Message: WidgetManager: Loading class IntSliderView:@jupyter-widgets/controls:2.0.0
13:09:25.288 [debug] Widget Message: WidgetManager: Loading class LayoutView:@jupyter-widgets/base:2.0.0
13:09:25.288 [debug] Widget Message: WidgetManager: Loading class StyleView:@jupyter-widgets/base:2.0.0
13:09:42.142 [debug] Start cell 3 execution @ undefined
13:09:42.143 [info] End cell 3 execution after 1706292565.207s, completed @ 1706292565207, started @ undefined
13:09:42.183 [info] Unhandled widget kernel message: stream [object Object]
Unhandled kernel message from a widget: stream : {"name":"stderr","text":"No such comm: d34adf99764f44519f1a8e7f42e1773e\n"}

Any suggestions?

**Comment by DonJayamanne** [comment]
Thanks for filling this issue and sorry you are running into this
Please can you provide the following information:
Versions if vscode, jupyter extensions, versions of python, steps to reproduce the error.

**Reply by DonJayamanne** [reply]
Closing in favour of https://github.com/microsoft/vscode-jupyter/issues/15082

Let's carry on from above issue
